src/main/scala/MoveMetrics.scala

```diff
@@ -0,0 +1,11 @@
+package chess
+
+case class MoveMetrics(
+    clientLag: Option[Centis] = None,
+    clientMoveTime: Option[Centis] = None
+) {
+
+  // Calculate client reported lag given the server's duration for the move.
+  def reportedLag(elapsed: Centis) =
+    clientMoveTime.fold(clientLag)(mt => Some(elapsed - mt nonNeg))
```

Comment:
this is `(elapsed - mt) nonNeg`, illustrating your point about why we should use parens to be clear